changelog/7744.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a bug where `~sunpy.map.sources.XRTMap` was still defaulting to `~astropy.units.ct` rather than
+`~astropy.units.DN`.
```

Comment:
```suggestion
Fixed a bug where `~sunpy.map.sources.XRTMap` was still defaulting to counts rather than DN.
```
<unknown>:9: WARNING: py:obj reference target not found: astropy.units.ct
<unknown>:9: WARNING: py:obj reference target not found: astropy.units.DN

Reply:
I knew that was going to happen.